Makefile

```
@@ -54,7 +54,8 @@ pull_xblock_translations:  ## pull xblock translations via atlas
 	python manage.py cms compile_xblock_translations
 
 clean_translations: ## Remove existing translations to prepare for a fresh pull
-	git clean -fdX conf/locale  # Removes core edx-platform translations but keeps Esperanto (eo) test translations
+	# Removes core edx-platform translations but keeps config files and Esperanto (eo) test translations
+	cd conf/locale && find . -type d -a ! -path . -a ! -path ./eo ! -path ./eo/* -exec rm -rf {} +
```

Please avoid using glob patterns in Makefile to avoid hitting Bash weird cases. The format below is more explicit and should be safer:

```suggestion
	find conf/locale -mindepth 1 -maxdepth 1 -type d -a ! -name eo -exec rm -rf {} +
```

---

Nice